client/state/selectors/test/get-new-application-password.js

```diff
@@ -0,0 +1,34 @@
+/** @format */
+
+/**
+ * Internal dependencies
+ */
+import { getNewApplicationPassword } from 'state/selectors';
+
+describe( 'getNewApplicationPassword()', () => {
+	test( 'should return the new application password if it exists', () => {
+		const newPassword = 'abcd 1234 efgh 5678';
+		const state = {
+			applicationPasswords: {
+				newPassword,
+			},
+		};
+		const result = getNewApplicationPassword( state );
+		expect( result ).toEqual( newPassword );
```

Comment:
You may have noticed I have a preference for strict `toBe` where possible 😅

Reply:
Done in https://github.com/Automattic/wp-calypso/pull/23907/commits/e4ea7cab2fd2dbf2765d11d5481611161f805ab1.

Also in https://github.com/Automattic/wp-calypso/pull/23907/commits/d18ad92aecfa98661ec552751f72579f9db324e7 removed the extra unnecessary test as suggested in https://github.com/Automattic/wp-calypso/pull/23906#discussion_r179365596.

Reply:
> removed the extra unnecessary test

😄 great!